.github/workflows/docs.yml

```diff
@@ -11,63 +11,167 @@
 # SPDX-License-Identifier: Apache-2.0
 # *******************************************************************************
 
-name: Documentation build
+name: Documentation
 on:
   pull_request:
-    types: [opened, reopened, synchronize]
+    types: [opened, reopened, synchronize, closed]
   push:
     branches:
       - main
```

Comment:
This filter needs to be removed.

Reply:
I meant the "branches: main" part, but ignore it, I just thought this was the reason the documentation build didn't run, but there has to be another reason.

Reply:
Incorrect indentation made the yaml file unparsable. It seems there is no error message in such cases!